src/com/android/mms/ui/SlideshowAttachmentView.java

```diff
@@ -101,8 +101,11 @@ public void setTextVisibility(boolean visible) {
     }
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit